src/tests/ftest/util/server_utils_params.py

```diff
@@ -507,9 +503,6 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
                 default_env_vars.extend(self.REQUIRED_ENV_VARS[name])
         self.env_vars = BasicParameter(None, default_env_vars)
 
-        # global CRT_CTX_SHARE_ADDR shared with client
-        self.crt_ctx_share_addr = BasicParameter(None, default_share_addr)
-
```

Comment:
Thanks. One more unused reference if you need to push again:
https://github.com/daos-stack/daos/blob/16ab7b5e679c55b868e602a1024940b49e70e197/src/tests/ftest/util/server_utils_params.py#L117